builder.py

```diff
@@ -247,6 +243,12 @@ def BuildQueue(self, queue):
                 queue.append('cd %s || exit $?' % self.path)
             queue.append('echo')
 
+        # Clean up the repostories of any existing (unnecessary files)
+        # We do this step here to properly handle any changes to .gitignore
+
+        queue.append('git clean -fdx')
+        queue.append('git submodule foreach git clean -fdx')
```

Comment:
This one is also superfluous, as `git clean` recurses from the top (which actually caused quite the problem in older versions of Git where submodules were stored in funky ways).